e2e/testdata/fn-render/path-index-ancestor/db/starlark-httpbin.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: fn.kpt.dev/v1alpha1
+kind: StarlarkRun
+metadata:
+  name: httpbin-gen
+source: |
+  httpbin_deployment = {
+     "apiVersion": "apps/v1",
+     "kind": "Deployment",
+     "metadata": {
+        "name": "httpbin",
+        "annotations": {
+          "config.kubernetes.io/index": 0,
+          "config.kubernetes.io/path": "../deployment_httpbin.yaml"
```

Comment:
@droot We need to fully specify the API here, it's part of the Functions Spec. Current docs:

https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/docs/api-conventions/config-io.md

This is ambiguous:

`When writing Resources to a directory, the Resource SHOULD be written to the corresponding path relative to that directory.`

We need to define which `directory` this refers to.

Reply:
Agree. Filed an issue to track this https://github.com/GoogleContainerTools/kpt/issues/1718

Reply:
How are we adding tests without knowing the expected behavior?